### PR TITLE
PHP-SDK for upi

### PR DESCRIPTION
**Pull request by ankitdas13**
Update Readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,19 @@ $refund = $api->refund->fetch($refundId); // Returns a particular refund
...
 
+//Upi
+$upi = $api->customer->create(array('name' => 'Razorpay User', 'email' => 'customer@razorpay.com')); // Creates an customer
+$upi = $api->order->create(array('receipt' => '123', 'amount' => 100, 'currency' => 'INR','customer_id'=>'cust_4xbQrmEoA5WJ01','method'=>'upi','token'=>array('max_amount'=>20000,'expire_at'=>1634215992,'frequency'=>'monthly'),'receipt'=>'Receipt No. 1')); // Creates an order
+$upi = $api->subscription->createSubscriptionRegistration(array('customer'=>array('name'=>'Gaurav Kumar','email'=>'gaurav.kumar@example.com','contact'=>'9123456780'),'type'=>'link','amount'=>100,'currency'=>'INR','description'=>'Registration Link for Gaurav Kumar','subscription_registration'=>array('method'=>'upi','max_amount'=>'500','expire_at'=>'1634215992'),'receipt'=>'Receipt No. 5','email_notify'=>1,'sms_notify'=>1,'expire_by'=>1634215992)); // Create a Registration Link 
+$upi = $api->invoice->fetch('inv_00000000000001')->notifyBy('sms'); // Send/Resend Notifications
+$upi = $api->payment->fetch($id); // Fetch Token by Payment ID
+$upi = $api->customer->fetch($customerId)->tokens()->fetch($tokenId); // Fetch Tokens by Customer ID
```
Comment (sonyy-m) [inline README.md:64]: Actually this is to fetch the particular token by customer id. I think you should add one more to fetch all tokens by customer Id.

File: src/Subscription.php [hunk 2]
```diff
@@ -32,4 +32,16 @@ public function createAddon($attributes = array())
...
+     * Create a Registration Link
+     * @param  array $attributes
+     * @return array
+     */
+    public function createSubscriptionRegistration($attributes = array())
+    {
+        $relativeUrl = 'subscription_registration/auth_links';
+        
+        return $this->request('POST', $relativeUrl, $attributes);
+    }
```
Comment (sonyy-m) [inline src/Subscription.php:46]: Please also check if you have done this function before.

File: src/Payment.php [hunk 3]
```diff
@@ -86,4 +86,11 @@ public function bankTransfer()
 
         return $this->request('GET', $relativeUrl);
     }
+
+    public function createRecurring($attributes = array())
+    {
+        $relativeUrl = $this->getEntityUrl() . 'create/recurring';
+ 
+        return $this->request('POST', $relativeUrl, $attributes);   
+    }
```
Comment (sonyy-m) [inline src/Payment.php:95]: I guess you already done this in cards module. If you have done this before will get Git Conflicts while merging.
So please check once.

Comment (ankitdas13) [reply src/Payment.php:95]: yes i have done this in card module but here i change the variable name

**Review by sonyy-m — APPROVED** [review]

**Pull request merged**